snippets.json

```diff
@@ -1,56 +1,1491 @@
+			"\t${5:disabled}",
+			"/>"
+		],
+		"description": "SegmentedControlItem"
+	},
+	"Select": {
+		"prefix": "NR1:Select",
+		"scope": "javascript,typescript",
+		"body": [
+			"// View more documentation at https://developer.newrelic.com/components/select/",
+			"<Select ",
+			"\tlabel=\"${1:Items}\" ",
+			"\t${2:labelInline} ",
+			"\tonChange={${3:this._onChange}}",
+			">",
+			"\t${4:SelectItem}",
+			"</Select>;"
+		],
+		"description": "Select"
+	},
+	"SelectItem": {
+		"prefix": "NR1:SelectItem",
+		"scope": "javascript,typescript",
+		"body": [
+			"// View more documentation at https://developer.newrelic.com/components/select-item/",
+			"<SelectItem value=\"${1:required}\">${2:string}</SelectItem>"
+		],
+		"description": "SelectItem"
+	},
+	"Steps": {
+		"prefix": "NR1:Steps",
+		"scope": "javascript,typescript",
+		"body": [
+			"// View more documentation at https://developer.newrelic.com/components/steps/",
+			"<Steps ",
+			"\tdefaultValue=\"${1:value}\" ",
+			"\tonChange={${2:this._onChange}}",
+			"\tspacingType={Steps.SPACING_TYPE.${3|EXTRA_LARGE,LARGE,MEDIUM,NONE,OMIT,SMALL|}}",
+			"\tvalue=\"${4:value}\"",
+			">",
+			"\t${5:StepsItem}",
+			"</Steps>"
+		],
+		"description": "Steps"
+	},
+	"StepsItem": {
+		"prefix": "NR1:StepsItem",
+		"scope": "javascript,typescript",
+		"body": [
+			"// View more documentation at https://developer.newrelic.com/components/steps-item/",
+			"<StepsItem",
+			"\tlabel=\"${1:required}\"",
+			"\tonClick={${2:this._onClick}}",
+			"\tvalue=\"${3:value}\"",
+			"\t${4:disabled}",
+			"\t${5:expanded}",
+			">",
+			"\t${6:content}",
+			"</Steps>"
+		],
+		"description": "StepsItem"
+	},
+	"Switch": {
+		"prefix": "NR1:Switch",
+		"scope": "javascript,typescript",
+		"body": [
+			"// View more documentation at https://developer.newrelic.com/components/switch/",
+			"<Switch",
+			"\tonChange={${1:this._onChange}}",
+			"\tlabel=\"${2:label}\"",
+			"\tinfo=\"${3:tooltip info}\"",
+			"\tspacingType={Switch.SPACING_TYPE.${4|EXTRA_LARGE,LARGE,MEDIUM,NONE,OMIT,SMALL|}}",
+			"\t${5:defaultChecked}",
+			"\t${6:disabled}",
+			"\t${7:invalid}",
+			"/>"
+		],
+		"description": "Switch"
+	},
+	"TextField": {
+		"prefix": "NR1:TextField",
+		"scope": "javascript,typescript",
+		"body": [
+			"// View more documentation at https://developer.newrelic.com/components/text-field/",
+			"<TextField ",
+			"\tlabel=\"${1:string}\" ",
+			"\tplaceholder=\"${2:string}\" ",
+			"\ttype={TextField.TYPE.${3|SEARCH,EMAIL,PASSWORD,PREFORMATTED,TEXT,URL|}}",
+			"\tinfo=\"${4:tooltip info}\" ",
+			"\tonChange={${5:this._onChange}}",
+			"\tspacingType={TextField.SPACING_TYPE.${6|EXTRA_LARGE,LARGE,MEDIUM,NONE,OMIT,SMALL|}}",
+			"\t${7:labelInline}",
+			"\t${8:disabled}",
+			"\t${9:autoFocus}",
+			"/>;"
+		],
+		"description": "TextField"
+	},
+	// Tables
```

Comment:
Unfortunately, comments aren't supported in JSON.

```suggestion
```

Reply:
VSCode allows C-style comments within snippet files! https://code.visualstudio.com/docs/editor/userdefinedsnippets#_create-your-own-snippets